packages/three-particle-emitter/src/index.ts

```diff
@@ -161,7 +161,8 @@ export class ParticleEmitter extends Mesh {
   }
 
   updateParticles() {
-    const planeGeometry = new PlaneBufferGeometry();
+    const texture = super.material ? (super.material as ShaderMaterial).uniforms.map.value: null;
```

Comment:
I wouldn't bother with this check for material since its always assigned in the constructor and a mesh is required to have a material and a geometry to be valid.

Reply:
Yeah, I'm pretty sure it's always going to have that material.